bokehjs/src/lib/models/glyphs/webgl/markers.ts

```diff
@@ -137,17 +131,12 @@ export abstract class MarkerGL extends BaseGLGlyph {
 
     // Upload data if we must. Only happens for main glyph.
     if (mainGlGlyph.data_changed) {
-      if (!(isFinite(trans.dx) && isFinite(trans.dy))) {
-        return  // not sure why, but it happens on init sometimes (#4367)
-      }
-      mainGlGlyph._baked_offset = [trans.dx, trans.dy]  // float32 precision workaround; used in _set_data() and below
       mainGlGlyph._set_data(nvertices)
+      if (this.glyph instanceof CircleView && this.glyph._radius != null) {
+        // Keep screen radius up-to-date for circle glyph. Only happens when a radius is given
+        this.vbo_s.set_data(0, new Float32Array(map(this.glyph.sradius, (s) => s*2)))
```

Comment:
`new Float32Array()` should be unnecessary, given that `sradius` is of the same type and `map()` is a type preserving transform.

Reply:
Well, I see the code is full of this kind of quirks, so this will have to be cleaned up at some point.

Reply:
Definitely, ultimately I would like to just always map ssize for circles as part of the "set data" work so that it can just be put straight into a buffer on the gl side (like everything else). I will remove the extra `Float32Array` here tho